platform/android/os_android.cpp

```diff
@@ -279,6 +292,15 @@ void OS_Android::set_keep_screen_on(bool p_enabled) {
 	godot_java->set_keep_screen_on(p_enabled);
 }
 
+void OS_Android::set_low_processor_usage_mode(bool p_enabled) {
+#ifdef TOOLS_ENABLED
+	// Disabled as it causes flickers. We also expect the devices running Godot in editor mode to be high end.
+	OS::set_low_processor_usage_mode(false);
```

Comment:
This is something that should be looked into eventually. Many users want to use low processor usage mode on Android and are having issues with it - and I do think it could be useful for the editor too, even on high end. You don't want the battery to be drained just because you had the Godot editor open to read its built-in documentation.

Should likely also call `OS_Unix` implementation in case it gets overridden in the future.